job-roles/solution-engineer

```diff
@@ -0,0 +1,29 @@
+Solution Engineer
```

Comment:
the file name should be solution-engineer.md otherwise it will not format properly.